src/pages/distroSettings/tabs/ProviderTab/schemaFields.ts

```diff
@@ -0,0 +1,40 @@
+import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
+
+const mergeUserData = {
+  type: "boolean" as "boolean",
+  title: "Merge with existing user data",
+};
+
+const securityGroups = {
+  type: "array" as "array",
+  title: "Security Groups",
+  items: {
+    type: "string" as "string",
+    title: "Security Group ID",
+    default: "",
+    minLength: 1,
+  },
+};
+
+const userData = {
+  type: "string" as "string",
+  title: "User Data",
+};
+
+export const staticProviderSettings = {
+  userData,
+  mergeUserData,
+  securityGroups,
+};
+
+export const uiSchema = {
+  providerSettings: {
+    "ui:ObjectFieldTemplate": CardFieldTemplate,
```

Comment:
nit. I think I might prefer this to be in `getFormSchema.ts` so that uiSchema only includes inputs that were defined in this file